src/creature.h

```diff
@@ -438,6 +438,11 @@ class Creature : public location, public viewer
         // accrue? mutates damage and pain
         virtual void deal_damage_handle_type( const effect_source &source, const damage_unit &du,
                                               bodypart_id bp, int &damage, int &pain );
+
+        // Pass handling bleed to creature/character
+        virtual void make_bleed( const effect_source &source, const bodypart_id &bp, time_duration duration,
+                                 int intensity = 1, bool permanent = false, bool force = false, bool defferred = false );
```

Comment:
This forces classes which inherit from this and use this function to define it, instead of using the dummy function which does nothing.
```suggestion
        virtual void make_bleed( const effect_source &source, const bodypart_id &bp, time_duration duration,
                                 int intensity = 1, bool permanent = false, bool force = false, bool defferred = false ) = 0;
```